InvalidConfigurationException when GCS store is not accessible on startup

**Describe the bug**

```
java.lang.IllegalStateException: Failed to execute CommandLineRunner
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:774) [spring-boot-3.1.2.jar:3.1.2]
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:755) [spring-boot-3.1.2.jar:3.1.2]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:319) [spring-boot-3.1.2.jar:3.1.2]
	at io.camunda.zeebe.broker.StandaloneBroker.main(StandaloneBroker.java:82) [camunda-zeebe-8.2.15.jar:8.2.15]
Caused by: io.camunda.zeebe.broker.system.InvalidConfigurationException: Failed configuring backup store GCS
	at io.camunda.zeebe.broker.system.SystemContext.validateBackupCfg(SystemContext.java:132) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.validateDataConfig(SystemContext.java:116) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.validateConfiguration(SystemContext.java:71) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.initSystemContext(SystemContext.java:60) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.<init>(SystemContext.java:56) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.StandaloneBroker.run(StandaloneBroker.java:87) ~[camunda-zeebe-8.2.15.jar:8.2.15]
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:771) ~[spring-boot-3.1.2.jar:3.1.2]
	... 3 more
Caused by: io.camunda.zeebe.backup.gcs.GcsBackupStoreException$ConfigurationException$CouldNotAccessBucketException: Bucket Expected to access bucket 'camunda-saas-int-backup-worker-3', but failed does not exist
	at io.camunda.zeebe.backup.gcs.GcsBackupStore.validateConfig(GcsBackupStore.java:200) ~[zeebe-backup-store-gcs-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.validateBackupCfg(SystemContext.java:124) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.validateDataConfig(SystemContext.java:116) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.validateConfiguration(SystemContext.java:71) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.initSystemContext(SystemContext.java:60) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.<init>(SystemContext.java:56) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.StandaloneBroker.run(StandaloneBroker.java:87) ~[camunda-zeebe-8.2.15.jar:8.2.15]
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:771) ~[spring-boot-3.1.2.jar:3.1.2]
	... 3 more
Caused by: com.google.cloud.storage.StorageException: Error getting access token for service account: oauth2.googleapis.com, iss: backups-worker-3@camunda-saas-int.iam.gserviceaccount.com
	at com.google.cloud.storage.StorageException.translate(StorageException.java:165) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.cloud.storage.spi.v1.HttpStorageRpc.translate(HttpStorageRpc.java:297) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.cloud.storage.spi.v1.HttpStorageRpc.list(HttpStorageRpc.java:441) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.cloud.storage.StorageImpl.lambda$listBlobs$11(StorageImpl.java:421) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.api.gax.retrying.DirectRetryingExecutor.submit(DirectRetryingExecutor.java:103) ~[gax-2.23.3.jar:2.23.3]
	at com.google.cloud.RetryHelper.run(RetryHelper.java:76) ~[google-cloud-core-2.13.0.jar:2.13.0]
	at com.google.cloud.RetryHelper.runWithRetries(RetryHelper.java:50) ~[google-cloud-core-2.13.0.jar:2.13.0]
	at com.google.cloud.storage.Retrying.run(Retrying.java:60) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.cloud.storage.StorageImpl.listBlobs(StorageImpl.java:418) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.cloud.storage.StorageImpl.list(StorageImpl.java:384) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at io.camunda.zeebe.backup.gcs.GcsBackupStore.validateConfig(GcsBackupStore.java:197) ~[zeebe-backup-store-gcs-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.validateBackupCfg(SystemContext.java:124) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.validateDataConfig(SystemContext.java:116) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.validateConfiguration(SystemContext.java:71) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.initSystemContext(SystemContext.java:60) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.<init>(SystemContext.java:56) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.StandaloneBroker.run(StandaloneBroker.java:87) ~[camunda-zeebe-8.2.15.jar:8.2.15]
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:771) ~[spring-boot-3.1.2.jar:3.1.2]
	... 3 more
Caused by: com.google.auth.oauth2.GoogleAuthException: Error getting access token for service account: oauth2.googleapis.com, iss: backups-worker-3@camunda-saas-int.iam.gserviceaccount.com
	at com.google.auth.oauth2.GoogleAuthException.createWithTokenEndpointIOException(GoogleAuthException.java:166) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.auth.oauth2.ServiceAccountCredentials.refreshAccessToken(ServiceAccountCredentials.java:543) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.auth.oauth2.OAuth2Credentials$1.call(OAuth2Credentials.java:269) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.auth.oauth2.OAuth2Credentials$1.call(OAuth2Credentials.java:266) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:?]
	at com.google.auth.oauth2.OAuth2Credentials$RefreshTask.run(OAuth2Credentials.java:633) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.common.util.concurrent.DirectExecutor.execute(DirectExecutor.java:31) ~[guava-31.1-jre.jar:?]
	at com.google.auth.oauth2.OAuth2Credentials$AsyncRefreshResult.executeIfNew(OAuth2Credentials.java:581) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.auth.oauth2.OAuth2Credentials.asyncFetch(OAuth2Credentials.java:232) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.auth.oauth2.OAuth2Credentials.getRequestMetadata(OAuth2Credentials.java:182) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.auth.oauth2.ServiceAccountCredentials.getRequestMetadata(ServiceAccountCredentials.java:938) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.auth.http.HttpCredentialsAdapter.initialize(HttpCredentialsAdapter.java:96) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.cloud.http.HttpTransportOptions$1.initialize(HttpTransportOptions.java:159) ~[google-cloud-core-http-2.13.0.jar:2.13.0]
	at com.google.cloud.http.CensusHttpModule$CensusHttpRequestInitializer.initialize(CensusHttpModule.java:109) ~[google-cloud-core-http-2.13.0.jar:2.13.0]
	at com.google.cloud.storage.spi.v1.HttpStorageRpc$InvocationIdInitializer.initialize(HttpStorageRpc.java:152) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.api.client.http.HttpRequestFactory.buildRequest(HttpRequestFactory.java:91) ~[google-http-client-1.43.1.jar:1.43.1]
	at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.buildHttpRequest(AbstractGoogleClientRequest.java:442) ~[google-api-client-2.2.0.jar:2.2.0]
	at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.executeUnparsed(AbstractGoogleClientRequest.java:552) ~[google-api-client-2.2.0.jar:2.2.0]
	at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.executeUnparsed(AbstractGoogleClientRequest.java:493) ~[google-api-client-2.2.0.jar:2.2.0]
	at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.execute(AbstractGoogleClientRequest.java:603) ~[google-api-client-2.2.0.jar:2.2.0]
	at com.google.cloud.storage.spi.v1.HttpStorageRpc.list(HttpStorageRpc.java:431) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.cloud.storage.StorageImpl.lambda$listBlobs$11(StorageImpl.java:421) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.api.gax.retrying.DirectRetryingExecutor.submit(DirectRetryingExecutor.java:103) ~[gax-2.23.3.jar:2.23.3]
	at com.google.cloud.RetryHelper.run(RetryHelper.java:76) ~[google-cloud-core-2.13.0.jar:2.13.0]
	at com.google.cloud.RetryHelper.runWithRetries(RetryHelper.java:50) ~[google-cloud-core-2.13.0.jar:2.13.0]
	at com.google.cloud.storage.Retrying.run(Retrying.java:60) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.cloud.storage.StorageImpl.listBlobs(StorageImpl.java:418) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.cloud.storage.StorageImpl.list(StorageImpl.java:384) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at io.camunda.zeebe.backup.gcs.GcsBackupStore.validateConfig(GcsBackupStore.java:197) ~[zeebe-backup-store-gcs-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.validateBackupCfg(SystemContext.java:124) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.validateDataConfig(SystemContext.java:116) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.validateConfiguration(SystemContext.java:71) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.initSystemContext(SystemContext.java:60) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.<init>(SystemContext.java:56) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.StandaloneBroker.run(StandaloneBroker.java:87) ~[camunda-zeebe-8.2.15.jar:8.2.15]
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:771) ~[spring-boot-3.1.2.jar:3.1.2]
	... 3 more
Caused by: java.net.UnknownHostException: oauth2.googleapis.com
	at sun.nio.ch.NioSocketImpl.connect(Unknown Source) ~[?:?]
	at java.net.SocksSocketImpl.connect(Unknown Source) ~[?:?]
	at java.net.Socket.connect(Unknown Source) ~[?:?]
	at sun.security.ssl.SSLSocketImpl.connect(Unknown Source) ~[?:?]
	at sun.net.NetworkClient.doConnect(Unknown Source) ~[?:?]
	at sun.net.www.http.HttpClient.openServer(Unknown Source) ~[?:?]
	at sun.net.www.http.HttpClient.openServer(Unknown Source) ~[?:?]
	at sun.net.www.protocol.https.HttpsClient.<init>(Unknown Source) ~[?:?]
	at sun.net.www.protocol.https.HttpsClient.New(Unknown Source) ~[?:?]
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(Unknown Source) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(Unknown Source) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(Unknown Source) ~[?:?]
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(Unknown Source) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getOutputStream0(Unknown Source) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(Unknown Source) ~[?:?]
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getOutputStream(Unknown Source) ~[?:?]
	at com.google.api.client.http.javanet.NetHttpRequest.execute(NetHttpRequest.java:113) ~[google-http-client-1.43.1.jar:1.43.1]
	at com.google.api.client.http.javanet.NetHttpRequest.execute(NetHttpRequest.java:84) ~[google-http-client-1.43.1.jar:1.43.1]
	at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1012) ~[google-http-client-1.43.1.jar:1.43.1]
	at com.google.auth.oauth2.ServiceAccountCredentials.refreshAccessToken(ServiceAccountCredentials.java:538) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.auth.oauth2.OAuth2Credentials$1.call(OAuth2Credentials.java:269) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.auth.oauth2.OAuth2Credentials$1.call(OAuth2Credentials.java:266) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:?]
	at com.google.auth.oauth2.OAuth2Credentials$RefreshTask.run(OAuth2Credentials.java:633) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.common.util.concurrent.DirectExecutor.execute(DirectExecutor.java:31) ~[guava-31.1-jre.jar:?]
	at com.google.auth.oauth2.OAuth2Credentials$AsyncRefreshResult.executeIfNew(OAuth2Credentials.java:581) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.auth.oauth2.OAuth2Credentials.asyncFetch(OAuth2Credentials.java:232) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.auth.oauth2.OAuth2Credentials.getRequestMetadata(OAuth2Credentials.java:182) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.auth.oauth2.ServiceAccountCredentials.getRequestMetadata(ServiceAccountCredentials.java:938) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.auth.http.HttpCredentialsAdapter.initialize(HttpCredentialsAdapter.java:96) ~[google-auth-library-oauth2-http-1.16.0.jar:?]
	at com.google.cloud.http.HttpTransportOptions$1.initialize(HttpTransportOptions.java:159) ~[google-cloud-core-http-2.13.0.jar:2.13.0]
	at com.google.cloud.http.CensusHttpModule$CensusHttpRequestInitializer.initialize(CensusHttpModule.java:109) ~[google-cloud-core-http-2.13.0.jar:2.13.0]
	at com.google.cloud.storage.spi.v1.HttpStorageRpc$InvocationIdInitializer.initialize(HttpStorageRpc.java:152) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.api.client.http.HttpRequestFactory.buildRequest(HttpRequestFactory.java:91) ~[google-http-client-1.43.1.jar:1.43.1]
	at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.buildHttpRequest(AbstractGoogleClientRequest.java:442) ~[google-api-client-2.2.0.jar:2.2.0]
	at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.executeUnparsed(AbstractGoogleClientRequest.java:552) ~[google-api-client-2.2.0.jar:2.2.0]
	at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.executeUnparsed(AbstractGoogleClientRequest.java:493) ~[google-api-client-2.2.0.jar:2.2.0]
	at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.execute(AbstractGoogleClientRequest.java:603) ~[google-api-client-2.2.0.jar:2.2.0]
	at com.google.cloud.storage.spi.v1.HttpStorageRpc.list(HttpStorageRpc.java:431) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.cloud.storage.StorageImpl.lambda$listBlobs$11(StorageImpl.java:421) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.api.gax.retrying.DirectRetryingExecutor.submit(DirectRetryingExecutor.java:103) ~[gax-2.23.3.jar:2.23.3]
	at com.google.cloud.RetryHelper.run(RetryHelper.java:76) ~[google-cloud-core-2.13.0.jar:2.13.0]
	at com.google.cloud.RetryHelper.runWithRetries(RetryHelper.java:50) ~[google-cloud-core-2.13.0.jar:2.13.0]
	at com.google.cloud.storage.Retrying.run(Retrying.java:60) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.cloud.storage.StorageImpl.listBlobs(StorageImpl.java:418) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at com.google.cloud.storage.StorageImpl.list(StorageImpl.java:384) ~[google-cloud-storage-2.20.2.jar:2.20.2]
	at io.camunda.zeebe.backup.gcs.GcsBackupStore.validateConfig(GcsBackupStore.java:197) ~[zeebe-backup-store-gcs-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.validateBackupCfg(SystemContext.java:124) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.validateDataConfig(SystemContext.java:116) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.validateConfiguration(SystemContext.java:71) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.initSystemContext(SystemContext.java:60) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.system.SystemContext.<init>(SystemContext.java:56) ~[zeebe-broker-8.2.15.jar:8.2.15]
	at io.camunda.zeebe.broker.StandaloneBroker.run(StandaloneBroker.java:87) ~[camunda-zeebe-8.2.15.jar:8.2.15]
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:771) ~[spring-boot-3.1.2.jar:3.1.2]
	... 3 more

```

Impact:
Fails broker startup. But restart succeeds.

**To Reproduce**

I don't think this is due to an actual configuration error, but because we validate GCS store configuration by trying to connect to the storage. If it is temporarily unavailable, validation fails. 

**Expected behavior**

Temporary errors should not cause validation failures.

**Log/Stacktrace**

[link to logs](https://console.cloud.google.com/errors/detail/CNeMl4mEyt_8NA;service=zeebe;time=P7D?project=camunda-saas-int)

**Environment:**
- Zeebe Version: 8.2.15

--- a/zeebe/backup-stores/gcs/pom.xml
+++ b/zeebe/backup-stores/gcs/pom.xml
@@ -83,6 +83,11 @@
     </dependency>
 
     <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
+
+    <dependency>
       <groupId>io.camunda</groupId>
       <artifactId>zeebe-backup-testkit</artifactId>
       <scope>test</scope>

--- a/zeebe/backup-stores/gcs/src/main/java/io/camunda/zeebe/backup/gcs/GcsBackupStore.java
+++ b/zeebe/backup-stores/gcs/src/main/java/io/camunda/zeebe/backup/gcs/GcsBackupStore.java
@@ -20,7 +20,7 @@ import io.camunda.zeebe.backup.api.BackupStore;
 import io.camunda.zeebe.backup.common.BackupImpl;
 import io.camunda.zeebe.backup.common.BackupStatusImpl;
 import io.camunda.zeebe.backup.common.Manifest;
-import io.camunda.zeebe.backup.gcs.GcsBackupStoreException.ConfigurationException.CouldNotAccessBucketException;
+import io.camunda.zeebe.backup.gcs.GcsBackupStoreException.ConfigurationException;
 import java.nio.file.Path;
 import java.util.Collection;
 import java.util.Optional;
@@ -28,15 +28,19 @@ import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.TimeUnit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public final class GcsBackupStore implements BackupStore {
   public static final String ERROR_MSG_BACKUP_NOT_FOUND =
       "Expected to restore from backup with id '%s', but does not exist.";
   public static final String ERROR_MSG_BACKUP_WRONG_STATE_TO_RESTORE =
       "Expected to restore from completed backup with id '%s', but was in state '%s'";
-  public static final String ERROR_MSG_ACCESS_FAILED = "Expected to access bucket '%s', but failed";
+  public static final String ERROR_VALIDATION_FAILED =
+      "Invalid configuration for GcsBackupStore: %s";
   public static final String SNAPSHOT_FILESET_NAME = "snapshot";
   public static final String SEGMENTS_FILESET_NAME = "segments";
+  private static final Logger LOG = LoggerFactory.getLogger(GcsBackupStore.class);
   private final ExecutorService executor;
   private final ManifestManager manifestManager;
   private final FileSetManager fileSetManager;
@@ -168,10 +172,16 @@ public final class GcsBackupStore implements BackupStore {
 
   public static void validateConfig(final GcsBackupConfig config) {
     try (final var storage = buildClient(config)) {
-      storage.list(config.bucketName(), BlobListOption.pageSize(1));
+      try {
+        storage.list(config.bucketName(), BlobListOption.pageSize(1));
+      } catch (final Exception e) {
+        LOG.warn(
+            "Unable to verify that the bucket %s exists, initialization will continue as it can be a transient network issue"
+                .formatted(config.bucketName()),
+            e);
+      }
     } catch (final Exception e) {
-      throw new CouldNotAccessBucketException(
-          ERROR_MSG_ACCESS_FAILED.formatted(config.bucketName()), e);
+      throw new ConfigurationException(ERROR_VALIDATION_FAILED.formatted(config), e);
     }
   }
 }

--- a/zeebe/backup-stores/gcs/src/test/java/io/camunda/zeebe/backup/gcs/ConfigIT.java
+++ b/zeebe/backup-stores/gcs/src/test/java/io/camunda/zeebe/backup/gcs/ConfigIT.java
@@ -8,7 +8,6 @@
 package io.camunda.zeebe.backup.gcs;
 
 import com.google.cloud.storage.BucketInfo;
-import io.camunda.zeebe.backup.gcs.GcsBackupStoreException.ConfigurationException.CouldNotAccessBucketException;
 import io.camunda.zeebe.backup.gcs.util.GcsContainer;
 import org.apache.commons.lang3.RandomStringUtils;
 import org.assertj.core.api.Assertions;
@@ -41,7 +40,7 @@ public class ConfigIT {
   }
 
   @Test
-  void shouldFailValidationIfBucketDoesNotExist() {
+  void shouldNotFailValidationIfBucketDoesNotExist() {
     // given
     final var bucketName = RandomStringUtils.randomAlphabetic(12);
     final var config =
@@ -52,8 +51,7 @@ public class ConfigIT {
             .build();
 
     // then
-    Assertions.assertThatThrownBy(() -> GcsBackupStore.validateConfig(config))
-        .isInstanceOf(CouldNotAccessBucketException.class)
-        .hasMessageContaining(config.bucketName());
+    Assertions.assertThatCode(() -> GcsBackupStore.validateConfig(config))
+        .doesNotThrowAnyException();
   }
 }

